scripts/install_libseccomp.sh

```diff
@@ -19,7 +19,7 @@ trap finish EXIT
 
 function build_and_install_gperf() {
     gperf_version="3.1"
-    gperf_url="https://ftp.gnu.org/gnu/gperf"
+    gperf_url="http://mirrors.kernel.org/gnu/gperf"
```

Comment:
https WFM, is there any reason to downgrade it?

Reply:
No reason, we can use https://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gperf instead of it

Reply:
@rusty-snake Could you give me your thoughts?

Reply:
I would go with `http_s_://mirrors.kernel.org/gnu/gperf`.

Reply:
Ok, (but https redirects http)